src/main/index.js

```diff
@@ -720,7 +720,8 @@ function runApp() {
   })
 
   ipcMain.handle(IpcChannels.GET_SYSTEM_LOCALE, () => {
-    return app.getLocale()
+    // we should switch to getPreferredSystemLanguages at some point and iterate through until we find a supported locale
```

Comment:
For the non-electron fallback we can use `navigator.languages` instead of `navigator.language`.